### ISSUE TITLE
Presence updates from unsubscribed contacts yield a notification

### ISSUE THREAD

**Issue by sternenseemann** [hannesm/jackline#144]
I have some buddy A and some buddy B.

I have properly added A, I am subscribed to A's presence updates and A is subscribed to my presence updates (`[_]` in the buddy bar).

B wasn't online since I have added B, so B is neither subscribed to me nor am I subscribed to B (`?_?` in the buddy bar).

From A, I receive such a XML message:

```xml
<presence type='unavailable' to='me' from='A'/>
```
From B, I receive such a XML message:

```xml
<presence type='unavailable' to='me' from='B'/>
```

When I `/connect`, the notification indicator for B is displayed, but not for A.

**Comment by cfcs** [comment]
Same, it is incredibly annoying. It also happens on each consecutive reconnect.

**Comment by hannesm** [comment]
sorry it took me several months, but I finally fixed this behaviour, together with #163 in current master.  presence messages from contacts not in your contact list will be displayed in your status log instead (and `/authorize allow <jid>` is now possible).  please tell if this solves the issue for you.  you may have some `~/.config/ocaml-xmpp-client/user/<B>` file you may need to remove to get rid of `<B>`.
